frontend/packages/vkt/src/components/publicExamEvent/listing/PublicExamEventListingRow.tsx

```diff
@@ -24,7 +24,7 @@ export const PublicExamEventListingRow = ({
   const isSelected = examEvent.id === selectedExamEvent?.id;
 
   const handleRowClick = () => {
-    dispatch(setOrToggleSelectedPublicExamEvent(examEvent));
+    !isPhone && dispatch(setOrToggleSelectedPublicExamEvent(examEvent));
```

Comment:
Rivin klikkaus ei olis tarkoitus olla deskarillakaan enää mahdollista. Tän dispatchin osion vois siirtää PublicExamEventDesktopCells alle jolloin pitäis toimia samoin kuin mobiilillakin

Reply:
Jeh, päivitetty ->